test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -209,18 +202,7 @@ class CloudFoundryDeployTest extends BasePiperTest {
     @Test
     void testCfNativeWithoutAppName() {
         helper.registerAllowedMethod('fileExists', [String.class], { s -> return true })
-        helper.registerAllowedMethod("readYaml", [Map], { Map m ->
-            if(m.text) {
-                return new Yaml().load(m.text)
-            } else if(m.file == 'test.yml') {
-                return [applications: [[]]]
-            } else if(m.file) {
-                return new Yaml().load((m.file as File).text)
-            } else {
-                throw new IllegalArgumentException("Key 'text' is missing in map ${m}.")
-            }
-        })
-
+        jryr.registerYaml('test.yml', { "applications: [[]]"} )
```

Comment:
I think `jryr.registerYaml('test.yml',  "applications: [[]]")` would be cleaner, but I have seen it is because throwing exception in another test. Maybe, we could handle that handle that case using a different approach, e.g. allowing both String and Closure as parameter or throwing the file not found exception in case the file was not registered.

Reply:
Of course we can also do this. I would like to suggest to change that in another pull request.